tools/pylint/del_method_checker.py

```diff
@@ -0,0 +1,51 @@
+from astroid import FunctionDef
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+
+from . import find_parent, ignore_tests
+
+ID_DEL_METHOD = "del-method-used"
+ID_EXCEPTION_IN_DEL = "exception-in-del"
```

Comment:
Do we have to enable these in the pylintrc? We have a `disable=all` in there.

Reply:
Added a commit to do this.